internal/app/handlers/auth.go

```diff
@@ -0,0 +1,29 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/DaniYer/GoProject.git/internal/app/middlewares"
+	"github.com/DaniYer/GoProject.git/internal/app/service"
+)
+
+func GetUserURLsHandler(svc *service.URLService) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		userID := r.Context().Value(middlewares.UserIDKey).(string)
```

Comment:
[SUGGESTION] It might be unsafe. Please add error checking to handle possible failures.